src/core_plugins/kibana/server/tutorials/apm/apm_server_instructions.js

```diff
@@ -71,7 +71,7 @@ export const WINDOWS_SERVER_INSTRUCTIONS = [
     textPre:
       '1. Download the APM Server Windows zip file from the [Download page](https://www.elastic.co/downloads/apm/apm-server).\n' +
       '2. Extract the contents of the zip file into `C:\\Program Files`.\n' +
-      '3. Rename the `apm-server-6.2-windows` directory to `APM-Server`.\n' +
+      '3. Rename the `apm-server-{config.kibana.version}-windows` directory to `APM-Server`.\n' +
```

Comment:
This is a good change, the windows zip file expands with the minor version, so `apm-server-6.2.3-windows-*`, not `apm-server-6.2-windows` as current listed.